.gitlab-ci.yml

```diff
@@ -14,6 +14,7 @@ variables:
 default:
   # Handle inputs when running as a downstream pipeline from the datadog-agent repo
   before_script:
+    - '[ -z "$MAJOR_VERSION" ] && export MAJOR_VERSION=7'
```

Comment:
nitpick, should the default value be saved as a variable, for more flexibility?

Reply:
that makes sense, I'll update